Game/Scripts/automation.py

```diff
@@ -0,0 +1,19 @@
+def onPGV(args):
+    startingPlayer()
+
+def deck_loaded(args):
+    if args.player == me:
+        me.setGlobalVariable("deckLoaded", "Yes")
+        if all(player.getGlobalVariable("deckLoaded") == "Yes" for player in getPlayers()):
+            twilight = table.create("fb909fa6-1ed7-44f7-a214-77b9bf5ce90d", -200, -31, 1, persist = True)

```

Comment:
A constant for the UUID would be nice, to it's more clear what this actually is.

Reply:
I'm not sure what you mean. I can comment it, but the table.create() function required a GUID to create a card. This function simply checks that all players have loaded a deck, then creates the Twilight Pool card on the table.